ocs_ci/templates/ocs-deployment/cluster.yaml

```diff
@@ -45,7 +45,7 @@ spec:
     # ssl: true
   network:
     # toggle to use hostNetwork
-    hostNetwork: false
+    hostNetwork: true
```

Comment:
Please see recent email(sub: Anyone using hostNetwork:true on OCP4?) from John Strunk. When he made hostNetwork: true and installed, the 'rook could not exec into the mons anymore'. Ceph cluster was not up.

It's better to wait for the fix for above issue

Reply:
Yep, replied to this mail thread yesterday.  And also set the Need deployment testing label for this with link to jenkins where we see it failed. This was mainly for test what Yaniv suggested in mail thread [HELP NEEDED] All ceph pods are disappearing, so I did this and in case it succeeded we could directly merge but as it doesn't work we cannot use this option without additional firewall configuration or wait for fix from the rook side.   @vasukulkarni  thanks, for settings WIP which will prevent us to merge this.